### PR TITLE
Display the smoothness slider correctly

### LINKED ISSUE #2248: Issue with plots: template: smooth
When building a plot with the smooth template and showing the results in VS-code the "smoothness indicator" shows on top of the plot(s) and cannot be modified. If I open the plots in the browser it works fine!

Screenshot 2022-08-24 at 12 00 50
Screenshot 2022-08-24 at 12 01 48

### PR DESCRIPTION
**Pull request by sroy3**
Closes #2248 

https://user-images.githubusercontent.com/3683420/186522954-a214ba75-9904-4183-87f0-2d7852f73872.mov

Try it yourself: https://5ffc93f46e9344002155f496-hvagdnhbph.chromatic.com/?path=/story/plots--smooth-template

### CODE REVIEW

**Comment by shcheklein** [comment]
Good fix, thanks @sroy3 . Can we control its color / position? Now it looks foreign a bit (clearly let's merge and ship this first).

**Comment by sroy3** [comment]
> Good fix, thanks @sroy3 . Can we control its color / position? Now it looks foreign a bit (clearly let's merge and ship this first).

Color, it's possible. I simply added a background, but did not try changing the default of the slider. Position is a little harder. Adding the slider into the plot rectangle makes the plot really small. Like you mentioned, I was thinking we should iterate on this after merging the fix.

**Comment by sroy3** [comment]
Screen Shot 2022-08-25 at 2 15 39 PM

With theme accent color

**Comment by sroy3** [comment]
Do not merge yet. Seems like there is a empty panel that pops up sometimes on other plots

File: webview/src/plots/components/templatePlots/TemplatePlotsGrid.tsx
Comment (mattseddon) [inline webview/src/plots/components/templatePlots/TemplatePlotsGrid.tsx:75]: [Q] What is the reason for recreating these?

**Review by mattseddon — APPROVED** [review]

File: webview/src/plots/components/templatePlots/TemplatePlotsGrid.tsx
Comment (sroy3) [reply webview/src/plots/components/templatePlots/TemplatePlotsGrid.tsx:75]: Since it's being created by vega the `vega-bindings` and that the `DragDropContainer` rebuilds the plots, setting these events on re-render is the only way to hook on them. Doing so once would create them too early. Maybe I can set up an observer or something to add the listeners once the elements are ready. I'll look into it and do a follow up if possible.

**Pull request merged**
